src/Repository/LinkBlockRepository.php

```diff
@@ -271,11 +270,11 @@ public function createTables()
         ];
 
         foreach ($queries as $query) {
-            $statement = $this->connection->executeQuery($query);
-
-            if ($statement instanceof Statement && 0 != (int) $statement->errorCode()) {
```

Comment:
https://github.com/doctrine/dbal/blob/3.6.x/UPGRADE.md#bc-break-errorcode-and-errorinfo-removed-from-connection-and-statement-apis